hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -525,6 +525,18 @@ public final class OzoneConfigKeys {
 
   public static final int OZONE_MANAGER_STRIPED_LOCK_SIZE_DEFAULT = 512;
 
+  public static final String OZONE_OM_FOLLOWER_READ_LOCAL_LEASE_ENABLED_KEY =
+      "ozone.om.follower.read.local.lease.enabled";
+  public static final boolean OZONE_OM_FOLLOWER_READ_LOCAL_LEASE_ENABLED_DEFAULT = false;
+
+  public static final String OZONE_OM_FOLLOWER_READ_LOCAL_LEASE_LAG_LIMIT =
+      "ozone.om.follower.read.local.lease.lag.limit";
+  public static final int OZONE_OM_FOLLOWER_READ_LOCAL_LEASE_LAG_LIMIT_DEFAULT = 30000;
+
+  public static final String OZONE_OM_FOLLOWER_READ_LOCAL_LEASE_TIME_MS =
+      "ozone.om.follower.read.local.lease.time.ms";
+  public static final long OZONE_OM_FOLLOWER_READ_LOCAL_LEASE_TIME_MS_DEFAULT = 15000;
```

Comment:
The defaults seem too big.  How about setting the lag to 1000 and the time to 3000ms (the same as OZONE_OM_RATIS_SERVER_REQUEST_TIMEOUT_DEFAULT)?

BTW, please move them to `OMConfigKeys`.